Engine/RealTime/BacktestingRealTimeHandler.cs

```diff
@@ -149,7 +149,7 @@ public void ScanPastEvents(DateTime time)
                     }
                     catch (ScheduledEventException scheduledEventException)
                     {
-                        var errorMessage = $"BacktestingRealTimeHandler.Run(): There was an error in a scheduled event {scheduledEvent.Name}. The error was {scheduledEventException.ScheduledEventExceptionMessage}";
+                        var errorMessage = $"BacktestingRealTimeHandler.Run(): There was an error in a scheduled event {scheduledEvent.Name}. The error was {scheduledEventException.Message}";
```

Comment:
The `Log.Error` below should be updated to accept the exception object as well, this will give us the full stack trace/ect...